### ISSUE TITLE
Y015 does not work with negative numbers and f-strings

### ISSUE THREAD

**Issue by Avasam** [PyCQA/flake8-pyi#286]
The following are flagged:
```py
POSITIVE = 1
STRING = ""
BYTES = b""
UNICODE = u""  # Also flagged by Y020
RAW = r""  # Also flagged by Y020
```

These are not:
```py
NEGATIVE = -1  # False negative
FSTRING = f"{0}"  # Flagged by Y020
FSTRING2 =f""  # Flagged by F541
```

**Comment by AlexWaygood** [comment]
As well as tweaking Y015 to catch the false-negatives you mention above, we should maybe exclude empty strings from Y020 — these aren't really "quoted annotations" if you're quoting "nothing".
